walkthroughs/ec2/ec2-cluster.yaml

```diff
@@ -0,0 +1,177 @@
+Description: >
+  This template deploys an EC2 instance to the provided VPC and subnets
+
+Parameters:
+
+  EnvironmentName:
+    Description: An environment name that will be prefixed to resource names
+    Type: String
+
+  InstanceType:
+    Description: Which instance type should we use to build the EC2 cluster?
+    Type: String
+    Default: c4.large
+
+  EC2AMI:
+    Description: The Amazon Machine Image ID used for the cluster, leave it as the default value to get the latest AMI
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
+#    Default: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
```

Comment:
we should drop these commented out lines

Reply:
Agree! I did and forgot to update. Thanks for catching.